mender-auth/main.cpp

```diff
@@ -51,6 +95,22 @@ int main(int argc, char *argv[]) {
 		}
 	}
 
+	auto ex_passphrase = GetPassphraseFromFile(passphrase_file);
```

Comment:
Remember to make it conditional, it's legal not to have a key.

Reply:
Giving the empty password (i.e., `""`) as an argument to `PrivateKey::LoadFromPEM()` is same as no password